tanner/emulators/sqli.py

```diff
@@ -28,21 +29,7 @@ def setup_db(self):
     @staticmethod
     @asyncio.coroutine
```

Comment:
It is not async function anymore. So, we should delete decorator and also change function call